vep_mapping_pipeline/run_consequence_mapping.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
+set -o pipefail
 
 # A wrapper script for running the consequence mapping pipeline concurrently and based on a VCF input.
 # See /README.md for detailed information.
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# Explanation of GNU parallel options:
+# --halt now,fail=1: if any job fails, kill the remaining ones immediately and report failure
+# --pipe: input is read from STDIN and split by chunks
+# -j 20: number of concurrent workers
+# -N 200: number of records (lines) per worker
 bcftools query "$1" -f '%CHROM:%POS:%REF:%ALT\n' \
   | sort -u \
-  | parallel --pipe -j 20 -N 200 python $DIR/consequence_mapping.py \
+  | parallel --halt now,fail=1 --pipe -j 20 -N 200 python3 $DIR/consequence_mapping.py \
```

Comment:
Given [the Travis CI specs](https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system) show a 2 core maximum, I am not sure we will be able to fully test the concurrency. But I think this is the best we can do at this time....

Reply:
Ah, but here's the thing: querying the VEP is not at all computationally intensive (on our side), and the number of jobs passed to parallel (`-j 20`) supersedes the actual number of CPUs present on the machine. So in Travis, we actually are starting 10 workers (not 20, because 2000 records in batches of 200 can only fill 10 workers with work) on a 2-core machine. This is fine because each worker spends the majority of its time waiting for VEP to return results